### DISCUSSION TITLE
[ 绳网 ] 手机端界面问题

### DISCUSSION THREAD

**Discussion by share121** [share121/inter-knot#1100] [其他]
关闭夜间模式，打开电脑模式

![image](https://github.com/user-attachments/assets/13197c1b-7ea0-4cf3-ac3f-95a50e613917)

**Comment by xiubachi** [comment]
好兄弟，求个梯子

**Comment by share121** [comment]
steam++ 就可以了

**Comment by hfdgcfgbcz** [comment]
手机能不能改回5列啊，4列看着有点难受

**Comment by zxc123qwer** [comment]
我靠，我终于进来了
